ci/release/changelogs/next.md

```diff
@@ -1,5 +1,7 @@
 #### Features 🚀
 
+- autoformat is supported on the CLI with the `fmt` subcommand.
```

Comment:
we should order changelogs by level of importance to user. i think Latex support is higher for example, maybe bump this down a few

Reply:
I very very strongly disagree. autofmt is much more important... latex is a subset of users, autofmt is literally everyone.

Reply:
We don't have any IDE plugins doing it automatically right now. So people will need to know about it in order to use it. And for plugin authors to implement it.